pkg/bundle/registry_client_test.go

```diff
@@ -70,9 +54,7 @@ func TestDownloadBundle(t *testing.T) {
 		ref := fmt.Sprintf("%s:%s-%s", baseRef, kubeVersion, tag)
 
 		_, err := ecr.DownloadBundle(ctx, ref)
-		if err == nil {
-			t.Errorf("expected error, got nil")
-		}
+		assert.EqualError(t, err, "pulling package bundle: test error")
```

Comment:
There exists [`assert.AnError`](https://pkg.go.dev/github.com/stretchr/testify/assert#pkg-variables) which can expression this kind of intention very well, plus it would let you use `assert.ErrorIs(t, assert.AnError)`, which is less brittle, as it doesn't depend on the text within the error.

Reply:
Yeh, it was easier to just dump the full text in. It would be nice if there was something like `assertErrorContains` to get a substring of the error